Source/UnitEx.cpp

```diff
@@ -921,3 +1116,80 @@ void UnitEx::SetAnimation(int animIdx, int animDelay, int animStartDelay, int bo
 
 	func(&(*unitArray)[unitID], 0, animIdx, animDelay, animStartDelay, boolInvisible, boolSkipDoDeath);
 }
+
+int UnitEx::GetNumTruckLoadsSoFar()
+{
+	if (!isInited) {
+		return HFLNOTINITED;
+	}
+
+	BeaconData* p = (BeaconData*)((int)(*unitArray) + (unitID*120) + 0x58);
+	return p->numTruckLoadsSoFar;
+}
+int UnitEx::GetBarYield()

```

Comment:
Would prefer to have spaces between each function closing bracket and the next function definition. This applies to several functions definitions in this PR. Could be done post-merge of this PR.